src/app/ApplicationTemplate.Shared.Views/Framework/Browser/BrowserService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Threading.Tasks;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.System;
+
+namespace ApplicationTemplate;
+
+public sealed class BrowserService : IBrowserService
+{
+	public async Task OpenAsync(Uri uri)
+	{
+		var launchSucceeded = await Launcher.LaunchUriAsync(uri);
+		if (!launchSucceeded)
```

Comment:
Shouldn't we throw an exception here ?

Reply:
May be, user would know what's going on.